joomla/joomla-cms
# Use alert-link class, change link color for default warning alert

#### Summary of Changes

Consistently use recently introduced `alert-link` class for all links in rendered alert divs.  Also, changes the color of alert links for the default warning alert div to match its color (and Bootstrap 3's implementation)

Before:
![screen shot 2016-07-19 at 3 25 17 pm](https://cloud.githubusercontent.com/assets/368545/16965305/2c36ede6-4dc5-11e6-8363-6e956f6218bc.png)

After:
![screen shot 2016-07-19 at 3 25 00 pm](https://cloud.githubusercontent.com/assets/368545/16965306/2d65cb2e-4dc5-11e6-8383-61397acbcff3.png)
#### Testing Instructions

In the update component, the link to the download page should now have the "bold" appearance.  Same for the link in the stats plugin's nag message and duplicate queries in the debug console's output.  Links in warning alerts (default yellow) should now have the brownish appearance instead of blue.

As for the stray CSS change with the `.categories-list .collapse` selector, that came after running the `generatecss.php` script.  No idea why else it'd be there.
